spec/features/all_families/admin_views_all_families_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "Admin can see all the families when in families#index" do
```

Comment:
A suggestion to change the title to be more accurate: "Admin sees all the families"

Reply:
Done in sha: `f92ad52`